### DISCUSSION TITLE
Unique URL for Testing Branch in Supabase?

### DISCUSSION THREAD

**Discussion by diegoserranobst** [supabase/supabase#23407] [Questions]
I'm in the process of setting up a separate testing environment for my project in Supabase and I plan to create a "testing" branch. 

My question is straightforward: When I create this testing branch, does Supabase automatically provide a new, unique URL for this branch? 

I need this to ensure that my frontend in the testing environment can operate independently of my production environment

**Comment by adlrwbr** [comment]
Hi Diego, did you ever figure this out? I'm stuck on the same thing!

**Reply by adlrwbr** [reply]
My solution
https://github.com/orgs/supabase/discussions/27939#discussioncomment-10057638
